java/query/query-lambda/src/main/java/sleeper/query/lambda/SqsQueryProcessorLambda.java

```diff
@@ -54,21 +53,26 @@ public class SqsQueryProcessorLambda implements RequestHandler<SQSEvent, Void> {
 
     private Instant lastUpdateTime;
     private InstanceProperties instanceProperties;
-    private final AmazonSQS sqsClient;
-    private final AmazonS3 s3Client;
-    private final AmazonDynamoDB dynamoClient;
+    private final SqsClient sqsClient;
+    private final S3Client s3Client;
+    private final DynamoDbClient dynamoClient;
+    private final S3TransferManager s3TransferManager;
     private QueryMessageHandler messageHandler;
     private SqsQueryProcessor processor;
 
     public SqsQueryProcessorLambda() throws ObjectFactoryException {
-        this(AmazonS3ClientBuilder.defaultClient(), AmazonSQSClientBuilder.defaultClient(),
-                AmazonDynamoDBClientBuilder.defaultClient(), System.getenv(CONFIG_BUCKET.toEnvironmentVariable()));
+        this(S3Client.create(), SqsClient.create(),
+                DynamoDbClient.create(),
+                S3TransferManager.builder().s3Client(S3AsyncClient.create()).build(),
```

Comment:
This should use the CRT async client with `S3AsyncClient.crtCreate`, see the AWS documentation:

https://docs.aws.amazon.com/sdk-for-java/latest/developer-guide/crt-based-s3-client.html